### ISSUE TITLE
Dock problem showing info while execute in remote

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot#48325]
**Godot version:**

3.3 mono stable

**OS/device including version:**

win10 64

**Issue description:**

The dock initially does not show information, later, even if a node is clicked, it does not show the information either.
The solution is to select another node and reselect the initial node.
This occurs while the game is running in remote mode

![behavior1](https://user-images.githubusercontent.com/19652075/116678892-2b3ec300-a9aa-11eb-87b6-c390f8ee9db4.gif)

**Steps to reproduce:**

**Minimal reproduction project:**

**Comment by Torguen** [comment]
Exists in v3.5.beta2.official [7a4f9dfb1]

**Comment by arkology** [comment]
@Torguen Wait, but this is the same as https://github.com/godotengine/godot/issues/33894...
FYI @akien-mga

**Comment by Torguen** [comment]
> [@Torguen](https://github.com/Torguen) Wait, but this is the same as [#33894](https://github.com/godotengine/godot/issues/33894)... FYI [@akien-mga](https://github.com/akien-mga)

Yes, I suppose so. Sometimes I forget that I’ve already reported a bug and report it again.
